src/QMCWaveFunctions/lcao/LCAOrbitalSet.h

```diff
@@ -48,7 +55,10 @@ namespace qmcplusplus
     vgl_type Temp; 
     ///Tempv(OrbitalSetSize) Tempv=C*Temp
     vgl_type Tempv; 
-
+    //lookup table mapping the unique determinants to their element position in C2_node vector
+    std::vector< std::vector<int> > lookup_tbl;
```

Comment:
This data seems MSD specific not generic for SPO.

Reply:
This is true. I was thinking that I would have overloaded functions for evaluatederivative and buildOptVariables specific for the single slater case (if that was your concern). Does that seem reasonable?

Reply:
You can have two dvaluatederivate interfaces for single and multi dets if that is needed.
buildOptVariable should be the same for single and mult dets.
If any data structure is needed by multi dets only, it needs to be moved to the MSD class and initialized in the MSD::buildOptVariables.
Here lookup_tbl doesn't seems needed for single determinant.

Reply:
@spinedaflores any comment on this?